data/events/2024/denver/main.yml

```
@@ -40,8 +40,6 @@ nav_elements: # List of pages you want to show up in the navigation of your page
  # - name: program
  # - name: speakers
   - name: sponsor
```

WDYT?  maybe leave this in as a comment, so it's easy to enable for next year
```suggestion
  - name: sponsor
 # - name: propose
 #   url: https://talks.devopsdays.org/devopsdays-denver-2024/cfp
```

---

I mean, next year you'll create a new event page, or is that not what you mean? Else: changes LGTM

---

doesn't the script to generate your new event rebuild all of this? 
also we need to hold on the PR right now anyway because parking details that are included are changing.